app/helpers/s3.py

```diff
@@ -62,19 +62,20 @@ def get_s3_file(wmts_path, etag=None):
         http_client = http.client.HTTPConnection(
             settings.AWS_BUCKET_HOST, timeout=settings.HTTP_CLIENT_TIMEOUT
         )
-        orig_connect = http.client.HTTPConnection.connect
-
-        def monkey_connect(self):
-            orig_connect(self)
-            # Set the following socket options
-            self.sock.setsockopt(socket.SOL_SOCKET, socket.SO_KEEPALIVE, 1)
-            self.sock.setsockopt(socket.SOL_TCP, socket.TCP_KEEPIDLE, 120)
-            self.sock.setsockopt(socket.SOL_TCP, socket.TCP_KEEPINTVL, 120)
-            return self
-
-        http_client.connect = monkey_connect(http_client)
-        http_client.request("GET", path, headers=headers)
-        response = http_client.getresponse()
+        try:
+            http_client.request("GET", path, headers=headers)
+            response = http_client.getresponse()
+        except ChunkedEncodingError as error:
+            # When reading from S3, sporadically TCP connections
+            # have been reset, raising 104 errors. We tried to active TCP
+            # keep alive to avoid such errors, but this did not help
+            logger.warning(
+                'TCP connection has been reset.' \
+                'when requesting file %s. error=%s',
+                wmts_path,
```

Comment:
I would provide the whole S3 path here, so when reading this log we will not be confused between WMS and S3 backend.